Assets/Scripts/Game/Tasks/TimerTask.cs

```diff
@@ -5,6 +5,7 @@ namespace Game.Tasks
     public abstract class TimerTask : GameTask
     {
         protected float remainingTime;
+        protected bool _timerStarted;
```

Comment:
I think the field should be declared as `private`, to not confuse anybody to change this value in the sub-classes.
Because the timer start condition should only be determined by the returned value of the `TimerStartCondition`.
Correct me, if i am wrong or misunderstood something.
```suggestion
        private bool _timerStarted;
```